kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -38,6 +39,20 @@ import scala.collection.JavaConverters._
 
 class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
 
+  test("CRC metadata & protocol fetching") {
+    withTempDir { dir =>
+      val path = dir.getCanonicalPath + "/table"
```

Comment:
you can just use the directory path for the table, we don't need to add a "/table"